src/main/java/bio/terra/janitor/service/stackdriver/StatsExporter.java

```diff
@@ -0,0 +1,34 @@
+package bio.terra.janitor.service.stackdriver;
+
+import bio.terra.janitor.app.configuration.StackdriverConfiguration;
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/** A component for setting up stackdriver stats exporting. */
+@Component
+public class StatsExporter {
+  private final Logger logger = LoggerFactory.getLogger(StatsExporter.class);
+
+  private final StackdriverConfiguration stackdriverConfiguration;
+
+  @Autowired
+  public StatsExporter(StackdriverConfiguration stackdriverConfiguration) {
+    this.stackdriverConfiguration = stackdriverConfiguration;
+  }
+
+  public void initialize() {
+    logger.info("Stackdriver stats enabled: {}.", stackdriverConfiguration.isEnabled());
+    if (!stackdriverConfiguration.isEnabled()) {
+      return;
+    }
+    try {
+      StackdriverStatsExporter.createAndRegister();
```

Comment:
If opencensus doesn't have support for automatic parsing of a version. One way to add a version tag to all metrics would be to pass an instance of `StackdriverStatsConfiguration` to `createAndRegister`.
The `setConstantLabels` method [here](https://www.javadoc.io/doc/io.opencensus/opencensus-exporter-stats-stackdriver/latest/io/opencensus/exporter/stats/stackdriver/StackdriverStatsConfiguration.Builder.html) could be used to automatically attach a version label to all metrics

Reply:
We talked about this offline. Right now, the image value has shown up at least once in stackdriver. Long term, we do want to be able to distinguish metrics by the image, but it'd be nice if we can include it as metadata on the resource instead of included in the metric labels. We should find a standard way to do this long term, and set images on each label if we have to.